src/EditorFeatures/CSharpTest/Diagnostics/UseAutoProperty/UseAutoPropertyTests.cs

```diff
@@ -244,5 +246,35 @@ public void TestWriteInNotInConstructor2()
 @"class Class { [|int i|]; public int P { get { return i; } } public Foo() { i = 1; } }",
 @"class Class { public int P { get; private set; } public Foo() { P = 1; } }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)]
+        public void TestAlreadyAutoPropertyWithGetterWithNoBody()
+        {
+            TestMissing(@"class Class { public int [|P|] { get; } }");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)]
+        public void TestAlreadyAutoPropertyWithGetterAndSetterWithNoBody()
+        {
+            TestMissing(@"class Class { public int [|P|] { get; set; } }");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)]
+        public void TestFieldWithAttributesCantBeMadeIntoAutoProperty()
+        {
+            TestMissing(@"
```

Comment:
Ahh, I didn't see that one and though I was covering a missed scenario.  I'll remove this test.